tests/integration/ceph_bucket_notification_test.go

```diff
@@ -47,20 +48,6 @@ func (s *ObjectSuite) TestBucketNotificationsInOrder() {
 	clusterInfo := client.AdminClusterInfo(namespace)
 	t := s.T()
 
-	t.Run("create CephObjectStoreUser", func(t *testing.T) {
```

Comment:
Removed this test case for cephobjectuser is not incorporated with bucket notification.